backend/application/dashboard/file/uploadFile/usecase.go

```diff
@@ -46,5 +42,11 @@ func (uc *UseCase) Execute(request *Request) (*Response, error) {
 		return nil, err
 	}
 
+	if err := uc.storage.Store(context.Background(), uuid, request.FileReader, request.Size); err != nil {
+		_ = uc.filesRepository.Delete(uuid)
```

Comment:
please assert for `filesRepository.AssertNotCalled(t, "Delete")`

Reply:
done